demo/test.c

```diff
@@ -1284,7 +1284,8 @@ static int test_mp_incr(void)
       goto LTM_ERR;
    }
    a.sign = MP_ZPOS;
-   if (mp_cmp_d(&a, MP_MASK) != MP_EQ) {puts("DDD");
+   if (mp_cmp_d(&a, MP_MASK) != MP_EQ) {
+      puts("DDD");
```

Comment:
DDD

Reply:
DDDaumen hoch

Reply:
To paraphrase @sjaeckel : I should have gone to bed a long time before that! ;-)

Reply:
> DDDaumen hoch